src/libponyrt/lang/stat.c

```diff
@@ -69,6 +71,8 @@ static void windows_stat(const char* path, pony_stat_t* p, struct __stat64* st,
   p->broken = false;
 
   p->hard_links = (uint32_t)st->st_nlink;
+  p->device = (uint32_t)st->st_dev;
```

Comment:
According to @dipinhora's [PR](https://github.com/ponylang/ponyc/pull/1575/files#diff-5334f48b2dfedb718c74dac1519824c9R19), the `st_dev` type should be `int64_t` on Linux, and `int32_t` on OSX.

Perhaps that means we should be using `int64_t` here (to be large enough for either system)?